xonsh/platform.py

```diff
@@ -210,13 +232,13 @@ def scandir(path):
     BASH_COMPLETIONS_DEFAULT = (
         '/usr/local/etc/bash_completion',
         '/opt/local/etc/profile.d/bash_completion.sh')
-elif ON_WINDOWS:
-    progamfiles = os.environ.get('PROGRAMFILES', 'C:/Program Files')
+elif ON_WINDOWS and GIT_FOR_WINDOWS_PATH:
```

Comment:
Yes. Before the defaults on windows was just hard-coded to the default location used by GItForWindows. If GitForWindows was available, it failed gracefully. 

Now the defaults are only added if we know GitForWindows is avaiable.